packages/nx-ignore/src/index.ts

```diff
@@ -90,12 +90,18 @@ async function main() {
   logDebug(`\n≫ Comparing ${baseSha}...${headSha}\n`);
 
   const graphJsonPath = join(tmpdir(), '.nx-affected-graph.json');
-  execSync(
-    `npx nx affected:graph --base=${baseSha} --head=${headSha} --file=${graphJsonPath}`,
-    {
-      cwd: root,
-    }
-  );
+  try {
+    execSync(
+      `npx nx affected:graph --base=${baseSha} --head=${headSha} --file=${graphJsonPath}`,
+      {
+        cwd: root,
+      }
+    );
+  } catch (e: any) {
+    if (e.stdout) console.error(e.stdout.toString());
+    if (e.stderr) console.error(e.stderr.toString());
+    exitWithoutBuild(`🛑 - Build cancelled due to the error above`);
```

Comment:
Mm... what if we exit WITH build? :thinking: 

`nx-ignore` isn't sure if it should or shouldn't be built...

We can either... say... we're not sure... it might be necessary though so... we'll build it anyways.

Or... we can say.... we're not sure.... you should probably fix the error so we are sure.... and then we'll build it.

Did you consider it?